src/sentry/api/endpoints/data_export.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
+
+
+from django.db import IntegrityError, transaction
+from rest_framework import serializers
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationEventPermission
+from sentry.api.serializers import serialize
+
+from sentry.models import ExportedData
+from sentry.tasks.data_export import compile_data
+
+
+class ExportedDataSerializer(serializers.Serializer):
+    query_type = serializers.IntegerField(
+        required=True, min_value=0, max_value=32767
```

Comment:
@leeandher The number `32767` seems arbitrary. Would be better to have the calculation here.